src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -29,9 +30,20 @@
     }
   },
   "packs": {
-    "Microsoft.Android.Sdk": {
+    "Microsoft.Android.Sdk.NET7": {
       "kind": "sdk",
       "version": "@WORKLOAD_VERSION@",
+      "alias-to": {
+        "osx-x64": "Microsoft.Android.Sdk.NET7.Darwin",
+        "osx-arm64": "Microsoft.Android.Sdk.NET7.Darwin",
+        "win-x86": "Microsoft.Android.Sdk.NET7.Windows",
+        "win-x64": "Microsoft.Android.Sdk.NET7.Windows",
+        "linux-x64": "Microsoft.Android.Sdk.NET7.Linux"
+      }
+    },
+    "Microsoft.Android.Sdk": {
+      "kind": "framework",
```

Comment:
I was getting two `AutoImport.props` one from .NET 6 and .NET 7, until I changed this to be a `framework`.

But we can still import this like:
```xml
<Import Project="Sdk.targets" Sdk="Microsoft.Android.Sdk" />
```

This feels weird to me, but gives the behavior we want? Basically we always use the .NET 7 `AutoImport.props`, but import the .NET 6 MSBuild targets for `net6.0-android` projects.

If we add new things in `AutoImport.props` that might break, we can condition them based on `TargetFrameworkVersion` being >= 7.0.